src/silx/gui/dialog/ColormapDialog.py

```diff
@@ -974,6 +974,12 @@ def __init__(self, parent=None, title="Colormap Dialog"):
         self._autoButtons = _AutoScaleButton(self)
         self._autoButtons.autoRangeChanged.connect(self._autoRangeButtonsUpdated)
 
+        self._saturation = qt.QSlider(qt.Qt.Horizontal, self)
```

Comment:
`_saturation` -> `_saturationSlider` ?

Reply:
What about using a QSpinBox instead?
Issue with silder is that the value is not visible.

A tooltip would be good as well since it is not necessarily obvious

Reply:
To me it's a detail which can be improved later.

But for a user point of view that's something you want to play with manually.

You can't play with a spinbox, it's boring.

Reply:
Yes, but you also want to see the value somehow

Reply:
I think I agree with both of you. So I think I will go for slider + ? QSpinBox ? or QLineEdit ...